test/new-e2e/tests/process/windows_test.go

```diff
@@ -292,14 +339,32 @@ func runDiskSpd(t *testing.T, remoteHost *components.RemoteHost) error {
 	// -r: Random I/O
 	// -Sh: Disable both software caching and hardware write caching.
 	// -w50: Write percentage
-	session, stdin, _, err := remoteHost.Start("diskspd -d120 -c128M -t2 -o4 -b8k -L -r -Sh -w50 disk-speed-test.dat")
+	cmd := []string{
+		"diskspd",
+		"-d120",
+		"-c128M",
+		"-t2",
+		"-o4",
+		"-b8k",
+		"-L",
+		"-r",
+		"-Sh",
+		"-w50",
+		"disk-speed-test.dat",
+	}
+	processName, err := runCMD(t, remoteHost, cmd)
+	return processName, cmd, err
+}
+
+func runCMD(t *testing.T, remoteHost *components.RemoteHost, cmd []string) (string, error) {
```

Comment:
```suggestion
func runWindowsCommand(t *testing.T, remoteHost *components.RemoteHost, cmd []string) (string, error) {
```

Suggestion: Rename because this is package-private, if it's specific to Windows then we should name it as such

Reply:
done!